src/generator.cs

```diff
@@ -717,6 +717,19 @@ public NamespaceManager (string prefix, string customObjCRuntimeNS, bool skipSys
 		if (Frameworks.HaveMetal && !(Generator.CurrentPlatform == PlatformName.MacOSX && !Generator.UnifiedAPI))
 			ImplicitNamespaces.Add (Get ("Metal"));
 
+		if (Frameworks.HaveCoreImage && !(Generator.CurrentPlatform == PlatformName.WatchOS))
```

Comment:
Why do you need the additional platform checks?

In the code above they're only there to not change the generated output when I ported the code to use IKVM; in theory those checks can be removed (but when porting to IKVM I preferred fixing one thing at a time, minimizing the diff in the generated output).